tasks/configure.yml

```diff
@@ -138,6 +138,48 @@
   with_items: "{{ haproxy_userlists }}"
   when: haproxy_userlists is defined
 
+## CREATE ERROR PAGES
+
+- name: Add error directory
+  file:
+    path: "{{ haproxy_config_dir }}/errors"
+    state: directory
+
+- name: '400'
+  copy:
+    src: 400.http
```

Comment:
I am not quite sure that yamllint let it go. I think you should provide quotes.

Reply:
yamllint doesn't complain with the default configuration.